sdk/batch.py

```diff
@@ -33,6 +33,7 @@ class Batch:
         - jobs: Dictionnary of all the jobs added to the batch.
         - jobs_count: number of jobs added to the batch
         - jobs_count_per_status: number of jobs per status
+        - slurm_job_id: The id of the job assigned by the Slurm scheduler (emulators only)
```

Comment:
Do we really want to expose that field to users ?

Reply:
@MatthieuMoreau0 There's an ongoing discussion on that side. Frankly, even though I added it, I think it's best not to expose it (it is pretty useless anyway for the users). If you agree, I am going to remove it.

Reply:
Yeah let's remove it! Don't forget to add it to excluded fields in the marshmallow schema in `public_apis`

Reply:
I agree, there is no use for the user in having access to this

Reply:
I think it was already removed from the public schema in public-apis, right ?

Reply:
@anneclairelh Yes, I confirm that this has already been done.